test/common/upstream/outlier_detection_impl_test.cc

```diff
@@ -77,6 +77,7 @@ TEST_F(OutlierDetectorImplTest, DetectorStaticConfig) {
   )EOF";
 
   Json::ObjectPtr custom_config = Json::Factory::LoadFromString(json);
+  EXPECT_CALL(*interval_timer_, enableTimer(std::chrono::milliseconds(100)));
```

Comment:
Just a missing EXPECT_CALL from a previous change. I added it here since innocuous. Can move to a separate PR if you prefer.